scripts/apps/users/components/TextWithMentions.tsx

```diff
@@ -12,23 +12,27 @@ const mentionRegexp = /@\[([^\]]+)\]\((desk|user):([^\)]+)\)/g;
  * @name TextWithMentions
  * @description Displays a text containing mentions.
  */
-export const TextWithMentions: React.StatelessComponent<any> = ({children, ...props}) => {
-    const msg = children;
-    const n = msg.length;
+
+export interface ITextWithMention {
+    message: string;
+    props: any;
```

Comment:
`props` isn't used anywhere, you should remove it. Also since we now have a TypeScript interface, we no longer need propTypes for this component, do remove it. Regarding naming, we have a convention to use `IProps`/`IState` for interfaces used for react component props/state - let's stick to that here too.